brave/src/main/java/brave/Tracing.java

```diff
@@ -34,10 +35,7 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.logging.Level;
-import java.util.logging.Logger;
-import zipkin2.reporter.AsyncReporter;
 import zipkin2.reporter.Reporter;
```

Comment:
Should this import be gone too?

Reply:
That being said, I think it's still fairly standard to import even from optional dependencies.

Reply:
no worries I'll revert it. this was accidental as I was trying to find where everything was called.